apps/accounts/views.py

```diff
@@ -161,7 +161,7 @@ def get(self, request, user, identity, next_path):
             return HttpResponseRedirect(path)
         else:
             login_user(request, user, identity)
-            next_path = next_path or '/'
+            next_path = next_path or reverse('home')
             log.info('Redirecting after login to: {}'.format(next_path))
             return HttpResponseRedirect(next_path)
```

Comment:
it would be great if we had a terminal function for this, like, `safe_redirect(next_path)`